Src/BootCamp.Chapter/Person.cs

```diff
@@ -1,7 +1,24 @@
-﻿namespace BootCamp.Chapter
+﻿using System;
+using System.Globalization;
+
+namespace BootCamp.Chapter
 {
     public class Person
     {
-        // add missing properties
+        public string Name { get; set; }
+        public string BirthDay { get; set; }
+        public string SureName { get; set; }
+        public string Gender { get; set; }
+        public string Country { get; set; }
+        public string Email { get; set; }
+        public string StreetAdress { get; set; }
+
+        public int CalculateAge()
```

Comment:
You probably wouldn't want to calculate age to be something you have to do often. Just do it once and store as a property. So you would want to call CalculateAge when you create the person object. If it was done via the constructor that would be tidier.

Reply:
Schould i change this or not.  Or schould we let Kai decide as the former remark that something could be a the constructor instead of a seperate function

Reply:
changed it because I like the code then better.

Reply:
@Matt500b age changes based on current date-time, therefore it should be recalculated every time. for example, you don't want the same age to remain after 5 years.